pkg/agent/proxy/proxier.go

```diff
@@ -346,6 +352,11 @@ func (p *proxier) uninstallLoadBalancerService(loadBalancerIPStrings []string, s
 }
 
 func (p *proxier) installServices() {
+	var serviceCIDRIPv4 *net.IPNet
```

Comment:
If this is only used in getMCSExportedServiceInfo, why not getting this within getMCSExportedServiceInfo?

Reply:
I feel it's unnecessary to put this into the loop of handling `p.serviceMap`.